### PR TITLE
renaming coda binary and coda-config

### PR DESCRIPTION
**Pull request by ghost-not-in-the-shell**

### CODE REVIEW

**Review by deepthiskumar — APPROVED** [review]

File: helm/archive-node/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: archive-node
 description: A Helm chart for Mina Protocol's archive node
 type: application
-version: 0.4.9
+version: 0.4.10
```
Comment (0x0I) [inline helm/archive-node/Chart.yaml:5]: Hey, can you also update the annotations below in Chart.yaml? The same description used in each CHANGELOG.md works. It's used to associate the version with a changelog in ArtifactHub (e.g. https://artifacthub.io/packages/helm/mina/archive-node?modal=changelog)

**Review by 0x0I — COMMENTED** [review]
LG2M - just a comment re: updating each Helm chart's ArtifactHub changelog annotations to keep things in sync.

File: helm/archive-node/Chart.yaml (hunk 1 above)
Comment (lk86) [inline helm/archive-node/Chart.yaml:5]: These changes are way beyond breaking, the new versions should be 1.0.0

File: helm/block-producer/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: block-producer
 description: A Helm chart for Mina Protocol's block producing network nodes
 type: application
-version: 0.5.2
+version: 0.5.3
```
Comment (lk86) [inline helm/block-producer/Chart.yaml:5]: These changes are way beyond breaking, the new versions should be 1.0.0

File: helm/seed-node/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: seed-node
 description: A Helm chart for Mina Protocol's seed nodes
 type: application
-version: 0.6.1
+version: 0.6.2
```
Comment (lk86) [inline helm/seed-node/Chart.yaml:5]: These changes are way beyond breaking, the new versions should be 1.0.0

File: helm/snark-worker/Chart.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: snark-worker
 description: A Helm chart for Mina Protocol's SNARK worker nodes
 type: application
-version: 0.4.8
+version: 0.4.9
```
Comment (lk86) [inline helm/snark-worker/Chart.yaml:5]: These changes are way beyond breaking, the new versions should be 1.0.0

**Review by lk86 — CHANGES_REQUESTED** [review]
Once the versions are bumped again (with artifacthub changelogs) it looks good to me

**Review by lk86 — APPROVED** [review]

**Review by 0x0I — APPROVED** [review]

**Comment by aneesharaines** [comment]
!approved-for-mainnet

**Pull request merged**